### PR TITLE
Add prettier .hbs writeup

### PR DESCRIPTION
**Pull request by amyrlam**
@jelhan sorry this came in kinda late. Thanks again for sharing, much appreciated! 

I read the discussion on #dev-ember-learning and just referred to them as Ember `.hbs` files for now.

### CODE REVIEW

File: content/the-ember-times-issue-173.md [hunk 1]
```diff
@@ -27,14 +28,15 @@ Check out the full [plan for the 3.0.0 release](https://github.com/ember-templat
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Prettier is coming to an Ember `.hbs` file near you! [Prettier](https://prettier.io/) is a popular opinionated code formatter. New Ember projects get Prettier out of the box for `.js` files, and existing projects can adopt Prettier with [`ember-cli-update`](https://cli.emberjs.com/release/basic-use/upgrading/).
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Check out [Jeldrik Hanschke (@jelhan)'s](https://github.com/jelhan) post [Format Glimmer templates with Prettier](https://dev.to/jelhan/format-glimmer-templates-with-prettier-ipa) and [tweet](https://twitter.com/jelhan1/status/1358537538932248576) to learn how to configure Prettier for Ember `.hbs` files, with specific tips to Prettier `.hbs` on save in VSCode. Note: Prettier for Glimmer templates does not (yet) respect meaningful whitespaces, which will be problematic if your application is not internationalized (i18n'd). Give it a try in your app to see if there are any gotchas.
```
Comment (jaredgalanis) [inline content/the-ember-times-issue-173.md]:
```suggestion
Check out [Jeldrik Hanschke (@jelhan)'s](https://github.com/jelhan) post [Format Glimmer templates with Prettier](https://dev.to/jelhan/format-glimmer-templates-with-prettier-ipa) and [tweet](https://twitter.com/jelhan1/status/1358537538932248576) to learn how to configure Prettier for Ember `.hbs` files, with specific tips to Prettier `.hbs` on save in VS Code. Note: Prettier for Glimmer templates does not (yet) respect meaningful whitespaces, which will be problematic if your application is not internationalized (i18n'd). Give it a try in your app to see if there are any gotchas.
```

**Review by jaredgalanis — APPROVED** [review]
Looks ✨ !

File: content/the-ember-times-issue-173.md [hunk 2]
```diff
@@ -27,14 +28,15 @@ Check out the full [plan for the 3.0.0 release](https://github.com/ember-templat
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Prettier is coming to an Ember `.hbs` file near you! [Prettier](https://prettier.io/) is a popular opinionated code formatter. New Ember projects get Prettier out of the box for `.js` files, and existing projects can adopt Prettier with [`ember-cli-update`](https://cli.emberjs.com/release/basic-use/upgrading/).
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Check out [Jeldrik Hanschke (@jelhan)'s](https://github.com/jelhan) post [Format Glimmer templates with Prettier](https://dev.to/jelhan/format-glimmer-templates-with-prettier-ipa) and [tweet](https://twitter.com/jelhan1/status/1358537538932248576) to learn how to configure Prettier for Ember `.hbs` files, with specific tips to Prettier `.hbs` on save in VS Code. Note: Prettier for Glimmer templates does not (yet) respect meaningful whitespaces, which will be problematic if your application is not internationalized (i18n'd). Give it a try in your app to see if there are any gotchas.
```
Comment (amyrlam) [inline content/the-ember-times-issue-173.md:36]:
```suggestion
<!-- alex ignore whitespaces -->
Check out [Jeldrik Hanschke (@jelhan)'s](https://github.com/jelhan) post [Format Glimmer templates with Prettier](https://dev.to/jelhan/format-glimmer-templates-with-prettier-ipa) and [tweet](https://twitter.com/jelhan1/status/1358537538932248576) to learn how to configure Prettier for Ember `.hbs` files, with specific tips to Prettier `.hbs` on save in VS Code. Note: Prettier for Glimmer templates does not (yet) respect meaningful whitespaces, which will be problematic if your application is not internationalized (i18n'd). Give it a try in your app to see if there are any gotchas.
```

**Comment by jelhan** [comment]
Thanks a lot for the great write up!

**Pull request merged**